deeplinkdispatch-processor/src/main/java/com/airbnb/deeplinkdispatch/DeepLinkProcessor.java

```diff
@@ -187,6 +187,7 @@ private void generateDeepLinkActivity() throws IOException {
         .endControlFlow()
         .addStatement("parameterMap.put(queryParameter, uri.getQueryParameter(queryParameter))")
         .endControlFlow()
+        .addStatement("parameterMap.put(DeepLink.URI, hostPath)")
```

Comment:
We should have an intact URI with the scheme, so we should replace `hostPath` with `uri.toString()`.